test/e2e/conf.js

```diff
@@ -32,6 +32,7 @@ if (process.env.TRAVIS  || process.env.CHROME_HEADLESS) {
     config.sauceKey = process.env.SAUCE_ACCESS_KEY;
   } else {
     // Desktop browser
+    config.directConnect = true;
```

Comment:
Need this while running on bacon. Fixes this error - https://github.com/angular/protractor/issues/4378